src/library_webgpu.js

```diff
@@ -2794,6 +2791,16 @@ var LibraryWebGPU = {
       "alphaMode": WebGPU.AlphaMode[
         {{{ gpu.makeGetU32('config', C_STRUCTS.WGPUSurfaceConfiguration.alphaMode) }}}],
     };
+
+    var viewFormatCount = {{{ gpu.makeGetU32('config', C_STRUCTS.WGPUSurfaceConfiguration.viewFormatCount) }}};
+
+    if (viewFormatCount) {
+      var viewFormats = {{{ makeGetValue('config', C_STRUCTS.WGPUSurfaceConfiguration.viewFormats, '*') }}};
+      // viewFormats pointer to an array of TextureFormat which is an enum of size uint32_t
+      configuration["viewFormats"] = Array.from({{{ makeHEAPView('32', 'viewFormats', `viewFormats + viewFormatCount * 4`) }}},
```

Comment:
Use single quotes here

Reply:
I just copied existing code from the same file. If you want the code to look different, feel free to modify the pr. To me, Java script is just gibberish that happens to sometimes do things. I'll leave all the rest up to you guys.

Reply:
Sorry it's inconsistent! Happy to take care of these things.

Reply:
This still won't be consistent but at least it's closer to the final target :)
```suggestion
      configuration['viewFormats'] = Array.from({{{ makeHEAPView('32', 'viewFormats', `viewFormats + viewFormatCount * 4`) }}},
```